aerogear/cordova-showcase-template
# AEROGEAR-2686 security check metrics

## Motivation

JIRA: https://issues.jboss.org/browse/AEROGEAR-2686

## Description

Integrate security check result metrics with the showcase app

## Verification steps

* Provision a metrics service
* Get the URL for the metrics service (from the metrics config map or however else you want)
* Update the metrics URL in `mobile-services.json` to the provisioned metrics service.
* Run the app
* Go to `chrome://inspect`
* Inspect the network tab of the running ionic app
* Open the security/device trust tab
* Confirm that network requests are made for each security check and that they don't fail
* Go to the Grafana instance URL that was created when provisioning the metrics service
* Open the Mobile Security Metrics tab and confirm that the metrics were persisted
